careers/students-and-graduates/index.html

```diff
@@ -0,0 +1,221 @@
+                    at the entry level for advanced degree candidates.
+                    The two-year appointment provides a fast-paced opportunity
+                    to gain experience and develop talents.
+                    Fellows will be challenged with opportunities
+                    to flourish into problem solvers, strategic thinkers, and future leaders.
+                </p>
+                <p class="short-desc">
+                    <a href="http://www.pmf.gov/" class="jump-link jump-link__right">
+                        Here's how to apply to be a Presidential Management Fellow
+                    </a>
+                </p>
+            </div>
+        </section>
+
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Job Application Process</h2>
+                <p class="short-desc">
+                    The application can be tricky.
+                    We can help.
+                    Learn how the application process works.
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn about job application process
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Working at the CFPB</h2>
+                <p class="short-desc">
+                    Help us continue to grow
+                    and develop our 21st century agency.
+                    The work you do will directly impact the lives of consumers
+                </p>
+                <p class="short-desc">
+                    <a href="#" class="jump-link jump-link__right">
+                        Learn what it's like to work at the CFPB
+                    </a>
+                </p>
+            </section>
+        </div>
+        <div class="block
+                    block__padded-top
+                    block__border-top
+                    content-l">
+            <section class="content-l_col-1-2">
+                <h2 class="h3">Follow us on LinkedIn</h2>
+                <p class="short-desc">
+                    The CFPB is one of the most searched for agencies
+                    in the federal government.
+                    Connect with us to stay updated on the work
+                    we do and new opportunities to be a part of it.
+                </p>
+                <p class="short-desc">
+                    <a href="https://www.linkedin.com/company/consumer-financial-protection-bureau"
+                       class="jump-link jump-link__right">
+                        Go to our LinkedIn
+                    </a>
+                </p>
+            </section>
+            <section class="content-l_col-1-2">
```

Comment:
These two need a vertical bar between them, however, it'll be a new style I'm pretty sure so don't worry about it I'm picking it up in another branch.